tuf/keys/db.go

```diff
@@ -58,6 +58,15 @@ func (db *KeyDB) AddRole(r *data.Role) error {
 	return nil
 }
 
+// GetAllRoles gets all roles from the database
+func (db *KeyDB) GetAllRoles() []*data.Role {
+	roles := []*data.Role{}
```

Comment:
Oops, nm!  I was confused and forgot `db.roles` was a map, since we were iterating over it and ignoring the key.  :)